pkgs/development/ocaml-modules/asn1-combinators/default.nix

```diff
@@ -1,31 +1,38 @@
 {
   lib,
   buildDunePackage,
-  fetchurl,
+  fetchFromGitHub,
   ptime,
   alcotest,
+  ohex,
 }:
 
 buildDunePackage rec {
-  minimalOCamlVersion = "4.08";
+  minimalOCamlVersion = "4.13.0";
 
   pname = "asn1-combinators";
-  version = "0.3.1";
+  version = "0.3.2";
 
-  src = fetchurl {
-    url = "https://github.com/mirleft/ocaml-asn1-combinators/releases/download/v${version}/asn1-combinators-${version}.tbz";
-    hash = "sha256-+imExupuHhxP4gM/AWWvYRljwkAM4roFEAS3ffxVfE4=";
+  src = fetchFromGitHub {
```

Comment:
This change is wrong.

Reply:
Sorry to bother you, but could you please tell me why you are using `fetchurl`, as there is still much I don't know about Nix, nixpkgs, and I would like to know why. The decompression results of the distributed tbz seem to match the source hosted on GitHub.